local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Tue May 12 18:23:39 BRT 2020
+sdk.dir=/Users/caroltr/Library/Android/sdk
```

Comment:
do we need this file?

Reply:
No we don't.
Gonna check gitignore file. Thanks!